README.md

```diff
@@ -182,6 +182,34 @@ module "eventbridge_with_permissions" {
 }
 ```
 
+### EventBridge with lambda rule schedule
+
+```hcl
+module "eventbridge" {
+  source = "terraform-aws-modules/eventbridge/aws"
+
+  create_bus = false
+
+  rules = {
+    orders = {
+      description = "Trigger for a Lambda"
```

Comment:
Please fix formatting in the code snippet, too.

Reply:
I changed the names according to what is in the full example and I didn't quite understand what should be fixed in lint. I followed the ECS example documentation that was already in the README.md file